### PR TITLE
BugherdTask_#642

### PR DESCRIPTION
**Pull request by dianaolives**
- I am still trying to figure out if the security token policy is just for single device registration or it includes bulk as well.
- I'm not sure if I should change the "To register devices" title, would have to search documentation for any references.

### CODE REVIEW

File: content/users-guide/device-management-bundle/connecting-devices.md [hunk 1]
```diff
@@ -193,7 +194,9 @@ The procedure of accepting devices is the same as described in [Optional securit
 
 While in this mode, any devices connecting to {{< product-c8y-iot >}} without a security token will be blocked and it won't be possible to complete their registration.
 
-#### To bulk-register devices
+<a name="bulk-registration"></a>
+#### Bulk device registration
+##### To bulk-register devices
 
 To connect larger amounts of devices, {{< product-c8y-iot >}} offers the option to bulk-register devices, that means, to register larger amounts of devices by uploading a CSV file.
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/connecting-devices.md:201]: I would move this section under the first heading (Bulk device registration), and start with the numbered list below the "To..." heading

File: content/users-guide/device-management-bundle/connecting-devices.md [hunk 2]
```diff
@@ -75,8 +75,9 @@ If you are subscribed to the required applications you will see other options fo
 Microservice developers can also use the [Extensible device registration](/concepts/applications/#extensible-device-registration) and implement a custom registration form that blends seamlessly into the UI.
 
 <a name="device-registration-manually"></a>
-#### To connect a  device manually
-
+#### Single device registration
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/connecting-devices.md:78]: I would add one more explanatory sentence here to separate the headings.

File: content/users-guide/device-management-bundle/connecting-devices.md (hunk 1 above)
Comment (dianaolives) [reply content/users-guide/device-management-bundle/connecting-devices.md:201]: Done

File: content/users-guide/device-management-bundle/connecting-devices.md (hunk 2 above)
Comment (dianaolives) [reply content/users-guide/device-management-bundle/connecting-devices.md:78]: Done

**Review by BeateRixen — APPROVED** [review]

**Review by rssag — APPROVED** [review]

**Pull request merged**